### PR TITLE
chore: adding tags on emr

### PR DESCRIPTION
**Pull request by jazibhumayun**
adding DE related tags on emr resources

### CODE REVIEW

File: batch/provision.yml [hunk 1]
```diff
@@ -32,6 +32,7 @@
     - bootstrap_actions: {}
     - steps: []
     - applications: []
+    - tags: []
```
Comment (jazibhumayun) [inline batch/provision.yml:35]: This should be overridden by analytics secure repo parameter [here](https://github.com/edx-ops/analytics-secure/pull/625/commits/f8f84909973ebac599ca03485e70c7c8064980c3)  but gives an error
`ParamValidationError(report=report.generate_report())\nParamValidationError: Parameter validation failed:\nInvalid type for parameter Tags[0], value:  [{u'Value': u'prod', type: <type 'str'>, valid types: <type 'dict'>\nInvalid type for parameter Tags[1], value:  u'Key': u'environment'}, type: <type 'str'>, valid types: <type 'dict'>\nInvalid type for parameter Tags[2], value:  {u'Value': u'analytics', type: <type 'str'>, valid types: <type 'dict'>\nInvalid type for parameter Tags[3], value:  u'Key': u'deployment'}, type: <type 'str'>, valid types: <type 'dict'>\nInvalid type for parameter Tags[4], value:  {u'Value': u'data-de', type: <type 'str'>, valid types: <type 'dict'>\nInvalid type for parameter Tags[5], value:  u'Key': u'cluster'}], type: <type 'str'>, valid types: <type 'dict'>\n"}`

**Review by estute — COMMENTED** [review]
So just to confirm, tags will be propagated from the master node into the cluster, right? If so, I think it is good to go.

**Comment by jazibhumayun** [comment]
> So just to confirm, tags will be propagated from the master node into the cluster, right? If so, I think it is good to go.

Yes that's correct. As per the [documentation](https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-plan-tags.html) `When you add a tag to an Amazon EMR cluster, the tag is also propagated to each active Amazon EC2 instance associated with the cluster. Similarly, when you remove a tag from an Amazon EMR cluster, that tag is removed from each associated active Amazon EC2 instance`

I actually checked that in aws account for currently running emr instances tags were propagating in underlying ec2 and ebs volumes

**Review by estute — APPROVED** [review]
LGTM

**Pull request merged**